### PR TITLE
Sound Synthesizer: I am growing stronger

### PR DESCRIPTION
**Pull request by ghost**
Oh yeah i thought it'd be funny if people thought there was an antag close so i did this since sound synths are rarely used 
the necksnap was the only one i had to tone down since it's loud as hell
:cl:
 * rscadd: Added some sounds to the sound synth the clown/mime gets

### CODE REVIEW

File: code/game/objects/items/devices/sound_synth.dm [hunk 1]
```diff
@@ -35,7 +35,15 @@
     "Welding Noises" = "selected_sound=sound/items/Welder.ogg&shiftpitch=1&volume=55",
 	"Quack" = "selected_sound=sound/items/quack.ogg&shiftpitch=1&volume=50",
 	"Short Slide Whistle" = "selected_sound=sound/effects/slide_whistle_short.ogg&shiftpitch=1&volume=50",
-	"Long Slide Whistle" = "selected_sound=sound/effects/slide_whistle_long.ogg&shiftpitch=1&volume=50"
+	"Long Slide Whistle" = "selected_sound=sound/effects/slide_whistle_long.ogg&shiftpitch=1&volume=50",
+	"Supermatter" = "selected_sound=sound/effects/suppermatter.ogg&shiftpitch=1&volume=80",
+	"Blood!" = "selected_sound=sound/effects/vampire_intro.ogg&shiftpitch=1&volume=75",
+	"Proboscis" = "selected_sound=sound/effects/lingextends.ogg&shiftpitch=1&volume=90",
+//	"Bwoink" = "selected_sound=sound/effects/adminhelp.ogg&shiftpitch=0&volume=100",
```
Comment (Probe1) [inline code/game/objects/items/devices/sound_synth.dm]: Never ever.

File: code/game/objects/items/devices/sound_synth.dm [hunk 2]
```diff
@@ -35,7 +35,15 @@
...
+//	"Malfunction" = "selected_sound=sound/effects/aimalf.ogg&shiftpitch=0&volume=100", too fun
+	"Time Stop" = "selected_sound=sound/effects/zaowlrd.ogg&shiftpitch=0&volume=90",
```
Comment (ghost) [inline code/game/objects/items/devices/sound_synth.dm]: smh anime references

**Comment by Pathid** [comment]
Isn't this
https://github.com/Commandersand/vgstation13/blob/ae25ac51cdbcc44d886ea0e99ace73b25e1c25c3/sound/effects/zaowlrd.ogg
the same effect as
https://github.com/Commandersand/vgstation13/blob/ae25ac51cdbcc44d886ea0e99ace73b25e1c25c3/sound/effects/theworld3.ogg

**Comment by DamianX** [comment]
image
definitely very similar
dumb anime posters

**Comment by jknpj** [comment]
the the duplicated za warudo sound

**Comment by Sooose** [comment]
Probably isn't a good idea to have clowns spamming "bwoink" and having people become desensitized to it in the middle of a round. Sure admins meme around with it a bit but that is usually either before or after the round.

File: code/game/objects/items/devices/sound_synth.dm (hunk 1 above)
Comment (ghost) [reply code/game/objects/items/devices/sound_synth.dm]: ye that's why i commented it out

**Comment by ghost** [comment]
ye i'll remove the bwoink and ai malf comments and remove the za warudo dupe
i also have to test the vampire sound because i think it didn't work properly

**Comment by ghost** [comment]
i'd also like to point out that the kirakrik sound is also a jojo reference i added a while ago

**Comment by Probe1** [comment]
Honestly this entire PR is a shit pile and we should not merge it.  You've turned a roundstart item into a knock off of advanced clown shoes + antag noises.  That's not good game design.

**Comment by ghost** [comment]
>clown shoes

you literally can't spam it
>antag noises

What's wrong with that? Instead of people instantly knowing the round type when they hear a noise they'll look around and check if a clown is being a dick head

**Comment by ghost** [comment]
replaced the vampire sound with YEET

**Comment by Probe1** [comment]
It devalues players experience by giving exclusive sounds to a synth.  It's the same reason why bwoink should never be added to an item by default.  You're messing with game design for easy humor.

**Comment by ghost** [comment]
>easy humor

maybe at first but people will notice that the clown can make those noises so they'll probably notice the clown with a black box in their hand when they hear a weird noise

**Comment by ghost** [comment]
K, now it's just the click, yeet, and BZZZT timestop sound

**Comment by Commandersand** [comment]
K, now it's just the click, yeet, and BZZZT timestop sound

**Comment by Probe1** [comment]
Codes fine and all the offending stuffs gone.

**Comment by ghost** [comment]
i was going to add a shitty mexican sound but ok holy fuck since when did PRs get merged so quickly

**Comment by Probe1** [comment]
Say it's a WIP if you aren't done.  You tee them up and I knock em down.

**Pull request merged**